File: openapi/openapi.json/code_samples/typescript/sessions_{sessionId}/get.ts
import { TestSdk } from 'test-sdk';

(async () => {
  const testSdk = new TestSdk({
    token: 'YOUR_TOKEN',
  });

  const { data } = await testSdk.qoSSessions.getSession('non q', {
    authorization: 'Authorization',
    xAuthorization: 'X-Authorization',
    xTransactionId: 'X-transaction-id',
  });

  console.log(data);
})();

File: openapi/code_samples/python/sessions_{sessionId}/get.py
from test_sdk import TestSdk, Environment

sdk = TestSdk(
    access_token="YOUR_ACCESS_TOKEN",
    base_url=Environment.DEFAULT.value,
    timeout=10000
)

result = sdk.qo_s_sessions.get_session(
    session_id="non q",
    authorization="Authorization",
    x_authorization="X-Authorization",
    x_transaction_id="X-transaction-id"
)

print(result)